mcs/class/System.Drawing/System.Drawing.Configuration/SystemDrawingSection.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Configuration;
+
+namespace System.Drawing.Configuration
+{
+    public sealed class SystemDrawingSection : ConfigurationSection
+    {
+        private const string BitmapSuffixSectionName = "bitmapSuffix";
```

Comment:
Please follow the http://www.mono-project.com/community/contributing/coding-guidelines/ in this file. I.e. no `private`, tabs instead of spaces, space before `(` etc.